src/nuclio/functions/version/version-monitoring/version-monitoring.less

```diff
@@ -77,6 +77,10 @@
                 }
             }
 
+            &.invocation-block {
```

Comment:
you used parent selector to the `.row` class. That means the class `invocation-block` isn't used